elements/barcode_info.go

```diff
@@ -0,0 +1,6 @@
+package elements
+
+type BarcodeInfo struct {
```

Comment:
Maybe BarcodeDimensions would be better name?

Reply:
ok changed